int ID
float confidence
geometry_msgs/Point position
geometry_msgs/Point velocity
geometry_msgs/Point acceleration
